lib/hydra/derivatives.rb

```diff
@@ -118,7 +118,7 @@ def constantize_processor(processor)
     end
 
     def transform_datastream(file_name, transform_directives, opts={})
```

Comment:
I don't see any test coverage for the opts (or for transform_datastream at all) do we know how it's supposed to be used? @stkenny: can you provide a use case and/or test for it?